pkg/catalogv2/git/download.go

```diff
@@ -18,13 +18,18 @@ import (
 const (
 	stateDir  = "management-state/git-repo"
 	staticDir = "/var/lib/rancher-data/local-catalogs/v2"
+	localDir  = "../rancher-data/local-catalogs/v2" // identical to helm.InternalCatalog
 )
 
 func gitDir(namespace, name, gitURL string) string {
 	staticDir := filepath.Join(staticDir, namespace, name, hash(gitURL))
 	if s, err := os.Stat(staticDir); err == nil && s.IsDir() {
 		return staticDir
 	}
+	localDir := filepath.Join(localDir, namespace, name, hash(gitURL))
```

Comment:
nit:
can you use a different variable name we are referencing a const variable and then mapping it to a local variable with the same name, which is a bit confusing

Reply:
`staticDir := filepath.Join(staticDir, namespace, name, hash(gitURL))`

This is exactly what the code in line 25 does. I'd rather be consistent with the surrounding code?